test/Elastic.Apm.Tests/BackendCommTests/PayloadSenderTests.cs

```diff
@@ -348,6 +348,20 @@ internal async void FlushInterval_test(TestArgs args, int numberOfEventsToSend)
 			}
 		}
 
+		[Fact]
+		public void Find_a_better_name()
+		{
+			var logger = new InMemoryBlockingLogger(LogLevel.Warning);
+			var configReader = new MockConfiguration(logger, apiKey: "*** INVALID_API_KEY ***");
+			var mockHttpMessageHandler = new MockHttpMessageHandler((r, c) => Task.FromResult(new HttpResponseMessage(HttpStatusCode.OK)));
```

Comment:
We'd need to return a content here, otherwise `ApmServerInfoProvider.cs` will fail already when reading the content [here](https://github.com/elastic/apm-agent-dotnet/pull/1787/files#diff-3f17e138c3b0c6eacad18b7496cc3ba19fe91dcd8e6d43269697ba9b28e41995R35).

Reply:
Agreed, a dedicated test file for `ApmServerInfo` makes more sense.